pom.xml

```diff
@@ -82,6 +82,24 @@
          </configuration>
        </plugin>
      </plugins>
+    <pluginManagement>
+      <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <archive>
+            <index>true</index>
+            <manifestEntries>
+              <Hudson-Version>1.609.18.1</Hudson-Version>
+              <Jenkins-Version>1.609.18.1</Jenkins-Version>
```

Comment:
>If you install it on something without the core update the reflection will not find the method and fall through to the old code path, so no worse than today.

= broken does not work....

So if you are going to accept broken "does not work" then dispense with the reflection shenanigans!